examples/aws/terraform/ha-autoscale-cluster/vars.tf

```
@@ -61,6 +61,12 @@ variable "route53_domain" {
   type = string
 }
 
+// Whether to set wildcard Route53
+// for use in Application Access
+variable "add_wildcard_route53_record" {
+  type = string
```

I think we should use `bool` instead of `string`. `bool` works for `TF_ENV_` too.
```suggestion
  type    = bool
  default = false
```